---a/net/minecraft/stats/Achievement.java
+++b/net/minecraft/stats/Achievement.java
@@ -6,131 +6,95 @@
 import net.minecraft.util.ChatComponentTranslation;
 import net.minecraft.util.EnumChatFormatting;
 import net.minecraft.util.IChatComponent;
-import net.minecraft.util.StatCollector;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class Achievement extends StatBase
-{
+
+public class Achievement extends StatBase {
+
     public final int displayColumn;
     public final int displayRow;
     public final Achievement parentAchievement;
     private final String achievementDescription;
-    @SideOnly(Side.CLIENT)
-    private IStatStringFormat statStringFormatter;
     public final ItemStack theItemStack;
     private boolean isSpecial;
     private static final String __OBFID = "CL_00001466";
 
-    public Achievement(String p_i46327_1_, String p_i46327_2_, int column, int row, Item p_i46327_5_, Achievement p_i46327_6_)
-    {
+    public Achievement(String p_i46327_1_, String p_i46327_2_, int column, int row, Item p_i46327_5_, Achievement p_i46327_6_) {
         this(p_i46327_1_, p_i46327_2_, column, row, new ItemStack(p_i46327_5_), p_i46327_6_);
     }
 
-    public Achievement(String p_i45301_1_, String p_i45301_2_, int column, int row, Block p_i45301_5_, Achievement p_i45301_6_)
-    {
+    public Achievement(String p_i45301_1_, String p_i45301_2_, int column, int row, Block p_i45301_5_, Achievement p_i45301_6_) {
         this(p_i45301_1_, p_i45301_2_, column, row, new ItemStack(p_i45301_5_), p_i45301_6_);
     }
 
-    public Achievement(String p_i45302_1_, String p_i45302_2_, int column, int row, ItemStack p_i45302_5_, Achievement parent)
-    {
+    public Achievement(String p_i45302_1_, String p_i45302_2_, int column, int row, ItemStack p_i45302_5_, Achievement parent) {
         super(p_i45302_1_, new ChatComponentTranslation("achievement." + p_i45302_2_, new Object[0]));
         this.theItemStack = p_i45302_5_;
         this.achievementDescription = "achievement." + p_i45302_2_ + ".desc";
         this.displayColumn = column;
         this.displayRow = row;
 
-        if (column < AchievementList.minDisplayColumn)
-        {
+        if (column < AchievementList.minDisplayColumn) {
             AchievementList.minDisplayColumn = column;
         }
 
-        if (row < AchievementList.minDisplayRow)
-        {
+        if (row < AchievementList.minDisplayRow) {
             AchievementList.minDisplayRow = row;
         }
 
-        if (column > AchievementList.maxDisplayColumn)
-        {
+        if (column > AchievementList.maxDisplayColumn) {
             AchievementList.maxDisplayColumn = column;
         }
 
-        if (row > AchievementList.maxDisplayRow)
-        {
+        if (row > AchievementList.maxDisplayRow) {
             AchievementList.maxDisplayRow = row;
         }
 
         this.parentAchievement = parent;
     }
 
-    public Achievement setIndependent()
-    {
+    public Achievement setIndependent() {
         this.isIndependent = true;
         return this;
     }
 
-    public Achievement setSpecial()
-    {
+    public Achievement setSpecial() {
         this.isSpecial = true;
         return this;
     }
 
-    public Achievement func_180788_c()
-    {
+    public Achievement func_180788_c() {
         super.registerStat();
         AchievementList.achievementList.add(this);
         return this;
     }
 
-    public boolean isAchievement()
-    {
+    public boolean isAchievement() {
         return true;
     }
 
-    public IChatComponent getStatName()
-    {
+    public IChatComponent getStatName() {
         IChatComponent ichatcomponent = super.getStatName();
         ichatcomponent.getChatStyle().setColor(this.getSpecial() ? EnumChatFormatting.DARK_PURPLE : EnumChatFormatting.GREEN);
         return ichatcomponent;
     }
 
-    public Achievement func_180787_a(Class p_180787_1_)
-    {
-        return (Achievement)super.func_150953_b(p_180787_1_);
+    public Achievement func_180787_a(Class p_180787_1_) {
+        return (Achievement) super.func_150953_b(p_180787_1_);
     }
 
-    @SideOnly(Side.CLIENT)
-    public String getDescription()
-    {
-        return this.statStringFormatter != null ? this.statStringFormatter.formatString(StatCollector.translateToLocal(this.achievementDescription)) : StatCollector.translateToLocal(this.achievementDescription);
-    }
 
-    @SideOnly(Side.CLIENT)
-    public Achievement setStatStringFormatter(IStatStringFormat p_75988_1_)
-    {
-        this.statStringFormatter = p_75988_1_;
-        return this;
-    }
-
-    public boolean getSpecial()
-    {
+    public boolean getSpecial() {
         return this.isSpecial;
     }
 
-    public StatBase func_150953_b(Class p_150953_1_)
-    {
+    public StatBase func_150953_b(Class p_150953_1_) {
         return this.func_180787_a(p_150953_1_);
     }
 
-    public StatBase registerStat()
-    {
+    public StatBase registerStat() {
         return this.func_180788_c();
     }
 
-    public StatBase initIndependentStat()
-    {
+    public StatBase initIndependentStat() {
         return this.setIndependent();
     }
-}
+}
